gdrive

```diff
@@ -57,16 +57,19 @@ def init():
     return service
 
 def get_filename(service, file_id):
-    """Gets the file name for a file ID"""
-    filename = 'gdrive-file'
+    """Gets the file name for a file ID and describes its info"""
+    filedata = None
     try:
-        file_metadata = service.files().get(fileId=file_id, fields='name').execute()
-        filename = file_metadata['name']
+        fields = 'id, name, size, modifiedTime, modifiedByMeTime, owners'
+        file_metadata = service.files().get(fileId=file_id, fields=fields).execute()
+        filedata = file_metadata
     except Exception as err:
-        print('An error happened while getting file name.')
-        print(err)
-
-    print('Filename is "%s" for ID: %s.' % (filename, file_id))
+        return False
```

Comment:
a function must not return either a boolean or a string. act as if it could only return one pre-defined type. maybe raise the exception instead of returning false.